packages/app/src/Navigator.tsx

```diff
@@ -82,6 +83,19 @@ function ProfileStack({
     );
 }
 
+const LiveTabsNavigator = createMaterialTopTabNavigator<LiveTabsParams>();
+
+export function LiveTabs(): ReactElement {
```

Comment:
NIT: In the future we are probably going to rename this. LiveTabs isn't very descriptive 😅